packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -306,6 +307,7 @@ String getNameForHostPlatform(HostPlatform platform) {
 }
 
 enum TargetPlatform {
+  android,
```

Comment:
maybe add comment about what the difference is between  `android`, `android_arm` and so on.

Reply:
we should probably just have `android` right?

Reply:
Added a comment on the usage of android vs android w/ arch. Refactoring this is similar to the iOS refactoring

Reply:
sg